webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/EndpointsDefinition.scala

```diff
@@ -179,27 +179,43 @@ object NcfApi extends ApiModuleProvider[NcfApi] {
 
   final case object UpdateTechnique extends NcfApi with ZeroParam with StartsAtVersion9 with SortIndex { val z = implicitly[Line].value
     val description = "Update technique created with technique editor"
-    val (action, path)  = POST / "techniques"
+    val (action, path)  = POST / "Internal" / "techniques"
   }
   final case object CreateTechnique extends NcfApi with ZeroParam with StartsAtVersion9 with SortIndex { val z = implicitly[Line].value
     val description = "Create a new technique in Rudder from a technique in the technique editor"
-    val (action, path)  = PUT / "techniques"
+    val (action, path)  = PUT / "internal" / "techniques"
   }
   final case object DeleteTechnique extends NcfApi with TwoParam with StartsAtVersion9 with SortIndex { val z = implicitly[Line].value
     val description = "Delete a technique from technique editor"
-    val (action, path)  = DELETE / "techniques" / "{techniqueId}" / "{techniqueVersion}"
+    val (action, path)  = DELETE / "internal" / "techniques"  / "{techniqueId}" / "{techniqueVersion}"
   }
   final case object GetResources extends NcfApi with TwoParam with StartsAtVersion15 with SortIndex { val z = implicitly[Line].value
     val description = "Get currently deployed resources of a technique"
-    val (action, path)  = GET / "techniques" / "{techniqueId}" / "{techniqueVersion}" / "resources"
+    val (action, path)  = GET / "internal" / "techniques" / "{techniqueId}" / "{techniqueVersion}" / "resources"
   }
   final case object GetNewResources extends NcfApi with TwoParam with StartsAtVersion15 with SortIndex { val z = implicitly[Line].value
-    val description = "Get ressources of a new technique"
-    val (action, path)  = GET / "techniques" / "{techniqueId}" / "new" / "{techniqueVersion}" / "resources"
+    val description = "Get resources of a technique draft"
+    val (action, path)  = GET / "internal" /  "techniques" / "draft" / "{techniqueId}" / "{techniqueVersion}" / "resources"
   }
   final case object ParameterCheck extends NcfApi with ZeroParam with StartsAtVersion15 with SortIndex { val z = implicitly[Line].value
     val description = "Get currently deployed resources of a technique"
-    val (action, path)  = POST / "techniques" / "parameter" / "check"
+    val (action, path)  = POST / "internal" / "techniques" / "parameter" / "check"
+  }
+  final case object GetTechniques extends NcfApi with ZeroParam with StartsAtVersion15 with SortIndex { val z = implicitly[Line].value
+    val description = "Get all Techniques metadata"
+    val (action, path)  = GET / "internal" / "techniques"
+  }
+  final case object GetMethods extends NcfApi with ZeroParam with StartsAtVersion15 with SortIndex { val z = implicitly[Line].value
+    val description = "Get all methods metadata"
+    val (action, path)  = GET / "internal" / "methods"
+  }
+  final case object UpdateMethods extends NcfApi with ZeroParam with StartsAtVersion15 with SortIndex { val z = implicitly[Line].value
```

Comment:
can you change version to 12? I think that since all is internal, version number should not change and use the last one.